### ISSUE TITLE
evaluating custom models

### ISSUE THREAD

**Issue by winniecook** [nanoporetech/bonito#413]
I have been fine-tuning the bonito dna_r10.4.1_e8.2_400bps_sup@v5.0.0 model and I want to test the performance of student vs teacher on my set of reads. How do I use this model as a basecaller to test the performance of student vs teacher?

**Comment by meesfrensel** [comment]
Something like this? Basecall the same set of reads with both models; compare basecalls to ground truth or a reference sequence; take the match rate as your performance indicator.

**Comment by davidnewman02** [comment]
> For the vast majority Oxford Nanopore sequencing use cases there is no need to train custom basecall models. Oxford Nanopore releases production basecall models through the [Dorado project](https://github.com/nanoporetech/dorado) and these are incorporated into the MinKnow device software available on all Oxford Nanopore devices. These models are trained using a highly diverse pool of training samples and rigorously validated to ensure robust performance across a wide range of sample types, making them the recommended choice for general use cases. https://github.com/nanoporetech/bonito/blob/master/documentation/training.md


The simplest way to test model performance is by basecalling with `bonito basecaller` and measuring alignment accuracy against a known reference and comparing that to the results of the released model. If you are training with a specific target in mind then you may wish to develop further tests for that application so you can assess any improvement in more detail.
